data/map planets.txt

```diff
@@ -1019,6 +1022,7 @@ planet "Ef Osch"
 planet Eldergrasp
 	attributes hai retirement station uninhabited
 	landscape land/station32
+	music ambient/machinery
 	description `A visitor to this station will immediately notice that the gravity is lower than most planets by about a third. There is a generously appointed reception area with an orderly on duty who informs you that this is a respite home for the elderly, and unless you have family here, you will not be permitted to wander the station.`
 	spaceport `The spaceport is virtually non-existent, with just a single cafeteria, a fuel pump for the desperate, and a dedicated loading area for loading and unloading the facility's regular deliveries. Despite its small size, everything is in excellent condition.`
```

Comment:
Should this station really have this sound?

Reply:
It still has gravity and houses people, so it should, yes. If we're concerned about the *specific* sound, we can worry about that later.

Reply:
Actually, now that I think of it, this station shouldn't really have this sound. It's not industrial or a military outpost, and the description says that it's well-maintained.